docker/Dockerfile

```diff
@@ -19,7 +19,7 @@
 # For those usages not covered by the GNU Affero General Public License
 # please contact with: [daniel.moranjimenez@telefonica.com]
 
-ARG NODE_VERSION=8.16.1-slim
+ARG NODE_VERSION=10.17.0-slim
```

Comment:
As other times (see https://github.com/telefonicaid/iotagent-ul/releases/tag/1.11.0) an entry in CHANGES_NEXT_RELEASE should be included about the changes, such as:

```
Upgrade NodeJS version from X.Y.Z to W.U.V in Dockerfile
```

Reply:
Fixed d44a5df